doc/sphinx/changes.rst

```diff
@@ -18,7 +18,99 @@ Version 9.1
+`Dune build system for OCaml and Coq/Rocq <https://github.com/ocaml/dune/>`_
+used to build the Rocq Prover itself and many Rocq projects.
+
+The `opam repository <https://github.com/coq/opam>`_ for Rocq packages has been maintained by
+Guillaume Claret, Guillaume Melquiond, Karl Palmskog, Matthieu Sozeau
+and Enrico Tassi with contributions from many users. The up-to-date list
+of packages is `available on the Rocq website <https://rocq-prover.org/packages>`_.
+
+Erik Martin-Dorel and Jaime Arias maintained the
+`Rocq Docker images <https://hub.docker.com/r/rocq/rocq-prover>`_.
+Erik Martin-Dorel maintained the `docker-keeper <https://gitlab.com/erikmd/docker-keeper>`_ compiler
+used to build and keep those images up to date (note that the tool is not Rocq specific).
+Erik Martin-Dorel and Théo Zimmermann maintained the
+`docker-coq-action <https://github.com/coq-community/docker-coq-action>`_
+container action (which is applicable to any opam project hosted on GitHub).
+
+Cyril Cohen, Vincent Laporte, Pierre Roux and Théo Zimmermann
+maintained the `Nix toolbox <https://github.com/coq-community/coq-nix-toolbox>`_.
+The docker-coq-action and the Nix toolbox are used by many Rocq projects for continuous integration.
+
+Rocq 9.1 was made possible thanks to the following 24 reviewers:
+Florian Angeletti, Ali Caglayan, Cyril Cohen, Pierre Courtieu, Jim
+Fehrle, Gaëtan Gilbert, Jason Gross, Emilio Jesús Gallego Arias,
+Jan-Oliver Kaiser, Thomas Lamiaux, Rodolphe Lepigre, Rodolphe Lepigre,
+Erik Martin-Dorel, Guillaume Melquiond, Patrick Nicodemus,
+Pierre-Marie Pédrot, Pierre Rousselin, Pierre Roux, Gabriel Scherer,
+Matthieu Sozeau, Nicolas Tabareau, Enrico Tassi, Théo Winterhalter and
+Théo Zimmermann.
+
+See the `Rocq Team <https://rocq-prover.org/rocq-team>`_ page for
+more details on Rocq's development teams.
+
+The 45 contributors to the 9.1 version are: JeanCASPAR, Soudant,
+ypopovitch, Reynald Affeldt, Wassim Ait-Moussa, David Allsopp,
+Christian Benedict Smit, Frédéric Besson, Mathis Bouverot, Ali
+Caglayan, Benedict Christian Smit, Cyril Cohen, Pierre Courtieu,
+Julien Cretin, Jian Fang, Jim Fehrle, Gaëtan Gilbert, Jason Gross,
+Dario Halilovic, Hugo Herbelin, Elyes Jemel, Emilio Jesús Gallego
+Arias, Jan-Oliver Kaiser, Kacper Korban, Lucie Lahaye, Thomas Lamiaux,
+Rodolphe Lepigre, Yann Leray, Kenji Maillard, Erik Martin-Dorel,
+Patrick Nicodemus, Charles Norton, Pim Otte, Pierre-Marie Pédrot,
+Josselin Poiret, Johann Rosain, Pierre Rousselin, Pierre Roux,
+Radosław Rowicki, Benedict Smit, Bastien Sozeau, Matthieu Sozeau,
+Nicolas Tabareau, Enrico Tassi and Théo Zimmermann.
+
+The Rocq community at large helped improve this new version via
+the GitHub issue and pull request system,
+the `Discourse forum <https://discourse.rocq-prover.org>`_ and the
+`Rocq Zulip chat <https://rocq-prover.zulipchat.com>`_.
+
+Gaëtan Gilbert and Pierre-Marie Pédrot are the release managers of Rocq 9.1.
```

Comment:
I removed the "Version X's development spanned Y months from the release of version Z." sentence because I don't know how the months should be counted (between releases? between branch dates? something else) and anyway it doesn't seem particularly useful.